herald/src/app/shared/player-data/player-data-store.component.ts

```diff
@@ -158,9 +175,10 @@ export class PlayerDataStore {
     * @returns          an observable which will return the data for the player requested
     */
     getPlayerData(playerName): Observable<SmallPlayerData> {
-      let playerUrl = `/assets/data/players/${playerName}.json`;
+      //let playerUrl = `/assets/data/players/${playerName}.json`;
 
-      return this.http.get(playerUrl)
+      //return this.http.get(playerUrl)//temporary removed whilst testing against real data
+      return this.http.get(`https://uthgard.org/herald/api/player/${playerName}`)
```

Comment:
tried building it as a service, the service wouldn't work with http, so I did it this way, the http is being passed in the constructor so it's still DI, just done manually